### PR TITLE
passing env var ANALYTICS_ENABLED to 'kusk install' command

### PR DESCRIPTION
**Pull request by jasmingacic**
This PR enables us to disable telemetry by running

```shell
ANALYTICS_ENABLED=false kusk install
```

Signed-off-by: jasmingacic <jasmin.gacic@gmail.com>

### CODE REVIEW

File: cmd/install.go [hunk 1]
```diff
@@ -224,10 +225,19 @@ func installKuskGateway(helmPath, releaseName, releaseNamespace string) error {
 		"--namespace",
 		releaseNamespace,
 		"--set", fmt.Sprintf("fullnameOverride=%s", releaseName),
+		"--set", os.Getenv("ANALYTICS_ENABLED"),
 		releaseName,
 		"kubeshop/kusk-gateway",
 	}
 
+	if enabled, ok := os.LookupEnv("ANALYTICS_ENABLED"); ok && enabled == "false" {
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply cmd/install.go:233]: missed it thanks

[the remark this answers is not in the document]
Comment (jasmingacic) [reply cmd/install.go:232]: the sequence of parameters is important

File: cmd/install.go [hunk 2]
```diff
@@ -228,6 +229,14 @@ func installKuskGateway(helmPath, releaseName, releaseNamespace string) error {
...
+		command = append(command, "")
+		copy(command[7:], command[6:])
+		command[6] = "--set"
+		copy(command[8:], command[7:])
+		command[7] = fmt.Sprintf("analytics.enabled=%s", enabled)
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply cmd/install.go:237]: Isn't it what we are doing here?

File: cmd/install.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (kylehodgetts) [reply cmd/install.go:232]: Yeah, this is a much better approach

**Review by kylehodgetts — APPROVED** [review]

**Pull request merged**